SnapWar/Auth/SWLoginStatusObserver.swift

```diff
@@ -0,0 +1,26 @@
+import SCSDKLoginKit
+
+class SWLoginStatusObserver: NSObject, SCSDKLoginStatusObserver {
```

Comment:
Where did you find the instructions to add this in?

Reply:
no explicit instructions, i just looked up the interface of the observer and read the comments.. the documentation is like non-existent

Reply:
@misspia you can use this to watch for login status changes, so you can actually register stuff with this observer if you need

Reply:
things that might be useful are to know when to show the bitmoji avatars and maybe logout buttons

Reply:
okay got it :D